exec/java-exec/src/main/java/org/apache/drill/exec/client/DrillClient.java

```diff
@@ -96,6 +96,8 @@
 
 import io.netty.channel.EventLoopGroup;
 
+import javax.validation.constraints.NotNull;
```

Comment:
Please use java.annotation.Nonnull or org.jetbrains.annotations.NotNull (or do not annotate, it is inferred).